### PR TITLE
Add PCIe primitive model

### PR DESCRIPTION
**Pull request by kowalewskijan**
This PR adds support for the PCIe primitive fasm2bels model.

### CODE REVIEW

**Review by litghost — CHANGES_REQUESTED** [review]
Change looks reasonable, but this needs a test (and likely a new prjxray-db)?

File: fasm2bels/models/pcie_models.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                pcie_site.add_source(pcie, port_name, wire_name, pcie.bel,
+                                     wire_name)
+
+    # Add the bel
+    pcie_site.add_bel(pcie)
+
+    # Add the sites
+    top.add_site(pcie_site)
+
+    top.disable_drc("REQP-47")
```
Comment (litghost) [inline fasm2bels/models/pcie_models.py:93]: This needs a comment

Comment (kowalewskijan) [reply fasm2bels/models/pcie_models.py]: Sorry, this is a leftover code - removed.

**Review by acomodi — APPROVED** [review]
LGTM

**Pull request merged**